central/sensor/service/connection/connection_test.go

```diff
@@ -170,6 +257,7 @@ func (s *testSuite) TestSendDeduperStateIfSensorReconciliation() {
 				sensorEventHandler: eventHandler,
 				sensorHello:        hello,
 				hashDeduper:        deduper,
+				scanSettingDS:      s.scanConfigDS,
```

Comment:
This shouldn't be needed in this test.

Reply:
When I remove the DS from the test it panics with `invalid memory address or nil pointer dereference`, I assume because it needs a DS to send the scanconfigs to.